lib/kwtool_web/router.ex

```diff
@@ -41,6 +41,9 @@ defmodule KwtoolWeb.Router do
     pipe_through [:browser, :auth, :ensure_auth]
 
     get "/home", HomeController, :index
+
+    get "/upload", UploadController, :index
+    post "/upload", UploadController, :create
```

Comment:
What do you think if we use plural form here -> `/uploads` because `get "/upload"` look a bit weird for listing.

Reply:
agree, updated.